vendor/engines/c2po/app/views/facility_accounts/account_fields/_credit_card_account.html.haml

```diff
@@ -1,4 +1,10 @@
-= f.input :description, :required => true, :hint => t('facility_accounts.account_fields.label.all_instruct'), :hint_html => { :class => 'help-inline' }
+- is_new ||= false
 
-= render :partial => 'facility_accounts/account_fields/affiliate', :locals => { :f => f }
-= render :partial => 'facility_accounts/account_fields/remittance_information', :locals => { :f => f }
\ No newline at end of file
+= f.input :description, required: true, hint: t('facility_accounts.account_fields.label.all_instruct'), hint_html: { class: 'help-inline' }
+= render partial: 'facility_accounts/account_fields/affiliate', locals: { f: f }
```

Comment:
Wasn't entirely sure where to put this note: when I load up one of the CC or PO forms, the "Affiliate Name" should be hidden. It should only appear once you've selected "Other". Seems like the JS is hooked up to the change event since changing the dropdown gives me correct behavior, it's ,just not on page load.